src/interfaces/store-interfaces/tombstone-interface.ts

```diff
@@ -1,5 +1,6 @@
 // Tombstone State model
 export interface TombStoneIF {
-  keycloakRoles: Array<string>,
+  keycloakRoles: Array<string>
   authRoles: Array<string>
+  entityType: string | null
```

Comment:
right on 👍  Good catch, i'm sure your already doing this but i know for myself i will have the strict settings on just while developing locally until we can get the fee summary playing nice!